.github/workflows/ci-cd.yml

```
@@ -77,4 +166,4 @@ jobs:
           aws ecs update-service \
             --cluster flaira \
             --service flairaprod \
-            --force-new-deployment
+            --force-new-deployment
```

YAMLlint (1.35.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)